### ISSUE TITLE
Running command always create new page

### ISSUE THREAD

**Issue by CyRamos** [EasyChris/obsidian-to-notion#52]
Hey
Whenever I run the command for 'share to notion' the workflow always creates a new entry for the page in notion) and generate new page each time instead of updating the same page. 
Thanks

**Comment by JOduMonT** [comment]
yes I noticed that too; but at least it doesnt create duplicate ;)
